pilot/pkg/model/wasm.go

```diff
@@ -17,41 +17,20 @@ package model
 import (
 	core "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
 	wasm "github.com/envoyproxy/go-control-plane/envoy/extensions/wasm/v3"
-
-	"istio.io/istio/pilot/pkg/features"
 )
 
-// ConstructVMConfig constructs a VM config. If WASM is enabled, the wasm plugin at filename will be used.
-// If not, the builtin (null vm) extension, name, will be used.
-func ConstructVMConfig(filename, name string) *wasm.PluginConfig_VmConfig {
-	var vmConfig *wasm.PluginConfig_VmConfig
-	if features.EnableWasmTelemetry && filename != "" {
```

Comment:
missed this env?

Reply:
Not sure what you mean?

Reply:
@hzxuzhonghu there's no need to have a special telemetry-wasm flag. Users should use Wasm API for this purpose.

Reply:
i mean this flag should also be removed

Reply:
oh thanks, missed that

Reply:
done